### PR TITLE
Apply body parser to server

### PR DESCRIPTION
**Pull request by JayChl**
- With body-parser we can parse incomming request

### CODE REVIEW

**Comment by nadongguri** [comment]
LGTM

File: server/server.js [hunk 1]
```diff
@@ -4,13 +4,20 @@
 
 const express = require('express');
 const path = require('path');
+const bodyParser = require('body-parser');
```
Comment (romandev) [inline server/server.js]: I think we should use |import| instead of |require|.

File: server/server.js [hunk 2]
```diff
@@ -4,13 +4,20 @@
...
 
 const config = require('./config');
 const httpsServer = require('./https_server/https_server');
 const redirectServer = require('./https_server/redirect_server');
 
 const app = express();
+
 app.use(express.static(path.join(__dirname, '../client')));
+app.use(bodyParser.json({ limit: '10mb' }));
```
Comment (romandev) [inline server/server.js]: I'm not sure if this limit is really needed in this branch.
IMHO, it's better to add this option if we really need this.
And if we need this right now, please leave a comment for this.

File: server/server.js [hunk 3]
```diff
@@ -4,13 +4,20 @@
...
+app.use(bodyParser.urlencoded({
```
Comment (romandev) [inline server/server.js]: ditto.

**Pull request merged**